lti_consumer/lti_xblock.py

```diff
@@ -257,6 +258,7 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
     """
 
     block_settings_key = 'lti_consumer'
+    i18n_js_namespace = 'LtiI18N'
```

Comment:
the make file already uses a name, please change this one
```suggestion
    i18n_js_namespace = 'XBlockLtiConsumerI18N'
```